ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/websocket/subscription/logs/LogsSubscriptionService.java

```diff
@@ -14,100 +14,40 @@
  */
 package org.hyperledger.besu.ethereum.api.jsonrpc.websocket.subscription.logs;
 
-import org.hyperledger.besu.ethereum.api.LogWithMetadata;
-import org.hyperledger.besu.ethereum.api.jsonrpc.internal.queries.BlockchainQueries;
-import org.hyperledger.besu.ethereum.api.jsonrpc.internal.queries.TransactionReceiptWithMetadata;
 import org.hyperledger.besu.ethereum.api.jsonrpc.internal.results.LogResult;
 import org.hyperledger.besu.ethereum.api.jsonrpc.websocket.subscription.SubscriptionManager;
 import org.hyperledger.besu.ethereum.api.jsonrpc.websocket.subscription.request.SubscriptionType;
 import org.hyperledger.besu.ethereum.chain.BlockAddedEvent;
 import org.hyperledger.besu.ethereum.chain.BlockAddedObserver;
 import org.hyperledger.besu.ethereum.chain.Blockchain;
-import org.hyperledger.besu.ethereum.core.Log;
 
 import java.util.List;
-import java.util.Optional;
 
 public class LogsSubscriptionService implements BlockAddedObserver {
 
   private final SubscriptionManager subscriptionManager;
-  private final BlockchainQueries blockchainQueries;
 
-  public LogsSubscriptionService(
-      final SubscriptionManager subscriptionManager, final BlockchainQueries blockchainQueries) {
+  public LogsSubscriptionService(final SubscriptionManager subscriptionManager) {
     this.subscriptionManager = subscriptionManager;
-    this.blockchainQueries = blockchainQueries;
   }
 
   @Override
-  public void onBlockAdded(final BlockAddedEvent event, final Blockchain blockchain) {
+  public void onBlockAdded(final BlockAddedEvent event, final Blockchain __) {
```

Comment:
What is the motivation for the underscore (`_`) as the input parameter name?

Reply:
I'ts to signify that the argument is not used